sierra_adapter/terraform/dynamo.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_dynamodb_table" "sierradata_table" {
+  name             = "SierraData_Merged"
+  read_capacity    = 1
+  write_capacity   = 1
+  hash_key         = "id"
+  range_key        = "version"
+  stream_enabled   = true
+  stream_view_type = "NEW_AND_OLD_IMAGES"
+
+  attribute {
+    name = "id"
+    type = "S"
+  }
+
+  attribute {
+    name = "version"
```

Comment:
Do we need the version to be a range key? There'll only ever be one version in the table right?